src/promptflow/promptflow/_cli/_pf/entry.py

```diff
@@ -94,6 +96,9 @@ def entry(argv):
 def main():
     """Entrance of pf CLI."""
     command_args = sys.argv[1:]
+    if len(command_args) == 1 and command_args[0] == 'version':
```

Comment:
we won't have help right? we can take care or this in further PR

Reply:
won't have for now.

will take care of this